indigo/signals.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.dispatch import receiver
+from background_task.signals import task_started, task_successful, task_error, task_finished
+import sentry_sdk
+
+
+# monitor background tasks
+@receiver(task_started)
+def bg_task_started(sender, **kwargs):
+    transaction = sentry_sdk.start_transaction(op="task")
```

Comment:
Does this error at all if sentry isn't configured?

Reply:
Since it's pretty standard to pull `signals.py` into an app, I don't think we should rely on that. Rather, assume it'll be imported and check within each signal handler if sentry is expected to be configured (eg. check settings for the DSN).

Reply:
Effectively transfer the DSN availability check from apps.py to the signals?

Reply:
Yes. Because otherwise we can't put any other handlers into `signals.py` that may not be related to sentry.